connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -152,11 +161,9 @@ class PlanGenerationTestSuite
   }
 
   private def readRelation(path: Path): proto.Relation = {
-    val input = Files.newInputStream(path)
-    try proto.Relation.parseFrom(input)
-    finally {
-      input.close()
-    }
+    val builder = proto.Relation.newBuilder()
+    parser.merge(Files.readString(path), builder)
```

Comment:
Are you sure it is safe to read this as a string? The files we are reading are binary (so no valid UTF-8).

Reply:
Oops, meant to revert this.